src/components/Tasks/Task_3/style.js

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet, } from 'react-native'
+import { screenSize, } from '../../../utils'
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    left: 0,
+    right: 0,
+    height: screenSize.height - 97,
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    paddingBottom: 10,
+    paddingHorizontal: 5,
+    backgroundColor: '#ffffff',
+  },
+})
```

Comment:
Duplicated style

Reply:
Yes. I have used Task_2 and Task_3 just to App will not fall down